client/src/lib/notifications/hooks/useTriggerNotification.spec.ts

```diff
@@ -11,6 +12,13 @@ const mockGetTriggerNotifications =
 const mockRequestPermission = notifee.requestPermission as jest.Mock;
 const mockCancelTriggerNotification =
   notifee.cancelTriggerNotification as jest.Mock;
+const mockAddEventListener = AppState.addEventListener as jest.Mock;
+
+beforeEach(() => {
```

Comment:
Why in `beforeEach`?

Reply:
Needed it since clearing all mocks made test fail, updated now to no before each and no after each

Reply:
Yeah I beleive it should have been `clearAllMocks` in `afterEach`.